easy_profiler_core/CMakeLists.txt

```diff
@@ -233,19 +233,12 @@ if (MSVC)
     target_compile_options(easy_profiler PRIVATE /WX)
 endif ()
 
-if (APPLE)
-    target_compile_options(easy_profiler PUBLIC -std=gnu++11)
-else ()
-    if (CMAKE_VERSION VERSION_LESS "3.1")
-        if (NOT MSVC)
-            target_compile_options(easy_profiler PUBLIC $<$<COMPILE_LANGUAGE:CXX>:-std=gnu++11>)
-        endif ()
-    else ()
-        if (NOT MSVC)
-            target_compile_options(easy_profiler PUBLIC -std=gnu++11)
-        endif ()
-        set_target_properties(easy_profiler PROPERTIES CXX_STANDARD 11 CXX_STANDARD_REQUIRED ON)
+if (CMAKE_VERSION VERSION_LESS "3.1")
+    if (NOT MSVC)
+        target_compile_options(easy_profiler PRIVATE $<$<COMPILE_LANGUAGE:CXX>:-std=gnu++11>)
```

Comment:
same than before, _except_ I made this flag `PRIVATE`.  Unfortunately, I currently do not know of any way to force a minimum C++ standard requirement to a library in cmake. However, leaving this to `PUBLIC` can cause more harm than good since this flag may conflict with user's c++14 and c++17 targets.